qcodes/tests/drivers/test_Keithley_2450.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import pytest
+
+from qcodes.instrument_drivers.tektronix.Keithley_2450 import Keithley2450, ParameterError
+
+import qcodes.instrument.sims as sims
+visalib = sims.__file__.replace('__init__.py', 'Keithley_2450.yaml@sim')
+
+
+@pytest.fixture(scope='module')
+def k2450():
```

Comment:
great to have these tests :)

Reply:
Thanks for the links, @astafan8. I was looking for an example notebook, which is why I could not find it